app/src/androidTest/java/com/duckduckgo/app/statistics/VariantManagerTest.kt

```diff
@@ -58,6 +58,29 @@ class VariantManagerTest {
         assertEqualsDouble(1.0, variant.weight)
         assertEquals(1, variant.features.size)
         assertTrue(variant.hasFeature(BottomBarNavigation))
+
+    }
+
+    // SERP Header Experiment
+    @Test
+    fun serpHeaderControlVariantHasExpectedWeightAndNoFeatures() {
+        val variant = variants.first { it.key == "zf" }
+        assertEqualsDouble(1.0, variant.weight)
+        assertEquals(0, variant.features.size)
+    }
+
+    @Test
+    fun serpHeaderVariantHasExpectedWeightAndSERPRemovalFeature() {
+        val variant = variants.first { it.key == "zg" }
+        assertEqualsDouble(1.0, variant.weight)
+        assertEquals(1, variant.features.size)
```

Comment:
Can you add a check that the feature is there? Checking the size only confirms that it has _a_ feature not the right one. When you test you'll notice you've mixed them up ""zg" _doesn't_ currently have the SERP removal feature.